src/actions/index.ts

```diff
@@ -0,0 +1,37 @@
+import build from './build';
+import move from './move';
+import attack from './attack';
+import transform from './transform';
+import catastrophe from './catastrophe';
+import sacrificeStart from './sacrificeStart';
+import sacrifice from './sacrifice';
+import concede from './concede';
+import endTurn from './endTurn';
+import chooseHomeworld from './chooseHomeworld';
+
+
+export default {
+  build,
+  move,
+  attack,
+  transform,
+  catastrophe,
+  sacrificeStart,
+  sacrifice,
+  concede,
+  endTurn,
+  chooseHomeworld
+};
+
+export {
+  build,
+  move,
+  attack,
+  transform,
+  catastrophe,
+  sacrificeStart,
+  sacrifice,
+  concede,
+  endTurn,
+  chooseHomeworld
+};
```

Comment:
Why is there an `export default` and an `export` which both include the same variables?